apps/manifest.json

```diff
@@ -21,6 +24,19 @@
             }
         }
     },
+    "chat-test": {
+        "context_name": "Chat UI test",
+        "menu_name": {
+            "appmenu.chat-test": {
+                "en": "Chat UI test",
+                "en-tts": "Chat UI test *EN*",
```

Comment:
Note: I need to remote the `*EN*` on en-tts locales.

Reply:
It is the case that sometimes `en-tts` needs some massaging because help messages that refer to visual cues are not valid for blind users. But yah, I think in general, there is no need to `*EN*` them, since it is going to be 99% correct as English.

Reply:
It's now fixed.